presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -37,6 +38,9 @@
 public class MySqlClient
         extends BaseJdbcClient
 {
+    // todo possibly make it configurable
+    public static final int MAX_IN_ROW_VARCHAR_LENGTH = 10000;
```

Comment:
This can be needed - as row is limited on 65535 bytes. And possibly we do not spend wholoe row on one or two long varchar fields.